commands/outliers.js

```diff
@@ -19,14 +19,18 @@ function * run (context, heroku) {
     ? 'CASE WHEN length(query) <= 40 THEN query ELSE substr(query, 0, 39) || \'…\' END'
     : 'query'
 
+  let limitString = /^(\d+)$/.exec(context.flags.limit)
+  let limit = limitString ? parseInt(limitString) : 10
```

Comment:
Does this work? The `exec` method [does not return a string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/exec)...

Reply:
great catch... i had it right when i ran in the console, but not when i translated it here.

Reply:
Huh well to answer my own question this does actually work thanks to JavaScript's absolute insanity:

```
> /^(\d+)$/.exec('1000')
[ '1000', '1000', index: 0, input: '1000' ]
> String(/^(\d+)$/.exec('1000'))
'1000,1000'
> parseInt('1000,1000')
1000
```